README_EN.md

```diff
@@ -0,0 +1,70 @@
+# KawaiiLogos
+
+Hello, good afternoon. This repository is where the logos created by Sawaratsuki have been uploaded.
+
+> [!WARNING]
+> The logos present here are created by Sawaratsuki, not by the services or entities they represent.
+> Not all are official logos in use.
+> Please be careful with too much "kawaii".
+
+> [!IMPORTANT]
+> You may not use the content of this repository in AI or anything Sawaratsuki considers equivalent to AI.
+> Please keep this in mind.
+
+## License
+
+This repository is licensed under the following custom license.
+
+### If you want to use a logo, please read this first
+
+### Definitions
+
+Here, "official" refers to the entities or persons who hold the rights to logos that were not created by Sawaratsuki, or any equivalent entity or organization.
+
+### General clauses
+
+- Personal use is free.
+  For example, sticker production falls under personal use.
+  > [!WARNING]
+  > You may not sell products created using these logos.
+  >
+- Commercial use is generally not allowed.
+  However, commercial use is possible in the following cases:
+  - If you wish to use a logo officially or have obtained permission in accordance with all clauses of this license from the official entity.
+  - If permitted under the "Each Logo License" below.
+  - If the official entity allows it.
+  - If "Licenses and specific notes for each logo" indicate that commercial use is possible.
+- It is not necessary to mention me or this repository on products created using this repository's logos, but doing so may motivate Sawaratsuki.
+- Sawaratsuki assumes no responsibility for damages arising from using the content of this repository.
+
+## Licenses and specific notes for each logo
+
+These are the licenses applied to each logo.
+In addition to the general clauses, please follow these licenses when using the logos.
+
+### Response code series
+
+> Codes like 404 NotFound, 403 Forbidden, 503 Service Unavailable, etc.,
+> They can be used on for-profit websites, among others.
```

Comment:
The "among others" here is ambiguous, and it wasn't worded like this in the original document as far as I can tell

Reply:
also now fixed! thanks!